src/lib.rs

```diff
@@ -208,8 +201,11 @@ impl Universe {
         self.cells = (0..self.width * height).map(|_i| Cell::Dead).collect();
     }
 
-    pub fn cells(&self) -> *const Cell {
-        self.cells.as_ptr()
+    pub fn cells(&self) -> js_sys::Uint8Array {
+        unsafe {
+            let u8_cells = mem::transmute::<&Vec<Cell>, &Vec<u8>>(&self.cells);
```

Comment:
I think this can avoid transmute here and instead cast the slice to pointer, convert the pointer, and construct a new slice from that:

```
    let u8_cells: &[u8] = unsafe {
        let ptr = self.cells.as_ptr() as *const u8;
        let bytelen = self.cells.len() * std::mem::size_of::<Cell>();
        std::slice::from_raw_parts(ptr, bytelen)
    };
```

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=98b83b545fce22aec8fdf4a5e2ed5c55